src/screens/FeedScreen/RenderList.js

```diff
@@ -146,7 +157,11 @@ const RenderListFeed = (props) => {
           source={source}
           headerStyle={styles.mh9}
           showAnonymousOption={showAnonymousOption}
-          onHeaderOptionClicked={onHeaderOptionClicked}
+          onDeletePost={onDeletePost}
+          isShowDelete={isShowDelete}
+          isSelf={isSelf}
+          isFollow={item?.is_following_target}
+          onPressFollUnFoll={() => followUnfollow(item)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

You've added new props and a function call to the `Header` component. Make sure these changes are reflected in the `Header` component's PropTypes and default props if necessary.

```diff
-          onHeaderOptionClicked={onHeaderOptionClicked}
+          onDeletePost={onDeletePost}
+          isShowDelete={isShowDelete}
+          isSelf={isSelf}
+          isFollow={item?.is_following_target}
+          onPressFollUnFoll={() => followUnfollow(item)}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->